Formula/c/cairo.rb

```diff
@@ -46,13 +41,6 @@ class Cairo < Formula
 
   uses_from_macos "zlib"
 
-  # Avoid segfaults on Big Sur. Remove at version bump.
-  # https://gitlab.freedesktop.org/cairo/cairo/-/issues/420
-  patch do
-    url "https://gitlab.freedesktop.org/cairo/cairo/-/commit/e22d7212acb454daccc088619ee147af03883974.diff"
-    sha256 "3b98004d7321c06d294fa901ac91964b6a4277ce4e53ef0cf98bf89e00d93332"
-  end
-
   def install
     args = %W[
```

Comment:
I missed this, but these arguments were actually not used. However, we do need to specify at least some of these build options. Many of the failures on macOS are with `_cairo_ft_font_face_create_for_pattern`, which requires `-Dfontconfig=enabled`. However, in this new Meson-based system it is [switched off on macOS unless explicitly enabled](https://gitlab.freedesktop.org/cairo/cairo/-/blob/1.18.0/meson.build?ref_type=tags#L260-L263).

Reply:
These may be required:

<details>
<summary> Diff </summary>

```diff
diff --git a/Formula/c/cairo.rb b/Formula/c/cairo.rb
index 959c376c07d..672938cf176 100644
--- a/Formula/c/cairo.rb
+++ b/Formula/c/cairo.rb
@@ -43,19 +43,18 @@ class Cairo < Formula
 
   def install
     args = %W[
-      --disable-dependency-tracking
-      --prefix=#{prefix}
-      --enable-gobject
-      --enable-svg
-      --enable-tee
-      --disable-valgrind
-      --enable-xcb
-      --enable-xlib
-      --enable-xlib-xrender
+      -Dfontconfig=enabled
+      -Dfreetype=enabled
+      -Dpng=enabled
+      -Dglib=enabled
+      -Dxcb=enabled
+      -Dxlib=enabled
+      -Dzlib=enabled
+      -Dglib=enabled
     ]
-    args << "--enable-quartz-image" if OS.mac?
+    args << "-Dquartz=enabled" if OS.mac?
 
-    system "meson", "setup", "build", *std_meson_args
+    system "meson", "setup", "build", *args, *std_meson_args
     system "meson", "compile", "-C", "build", "--verbose"
     system "meson", "install", "-C", "build"
   end
```
</details>

Reply:
Thanks for pointing that out.
I had not noticed that the args were no longer in use either.

Fixed. https://github.com/Homebrew/homebrew-core/pull/145154/commits/c1176dea3b32acf40c1fda96b2903ae18feb629b